src/opentype.mjs

```diff
@@ -533,51 +444,20 @@ function parseBuffer(buffer, opt={}) {
  * We use the node.js callback convention so that
  * opentype.js can integrate with frameworks like async.js.
  * @alias opentype.load
- * @param  {string} url - The URL of the font to load.
- * @param  {Function} callback - The callback.
+ * @deprecated
  */
-function load(url, callback, opt = {}) {
```

Comment:
I thought we agreed that this would still function in 2.0.0 with a warning, and be removed in a subsequent release.

Reply:
I thought the function would be *replaced* with a deprecation message